meta/downloadHelperBinaries.ts

```diff
@@ -108,7 +108,7 @@ async function downloadLegendary() {
 }
 
 async function downloadGogdl() {
-  return downloadGithubAssets(
+  await downloadGithubAssets(
```

Comment:
I understand these were `return` on purpose cause it's awaiting all promises later

https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/blob/829c8dc68d3251f4eaa75de1d0f70f80678ccbaf/meta/downloadHelperBinaries.ts#L220

Reply:
Those functions return a Promise themselves this doesn't change anything really
Here, I'm printing a date before and after awaiting promises

```
Downloading: [ 'legendary', 'gogdl', 'nile', 'comet' ]
Downloading legendary for linux x64 from https://github.com/Heroic-Games-Launcher/legendary/releases/download/0.20.35/legendary_linux_x86_64
Downloading legendary for darwin x64 from https://github.com/Heroic-Games-Launcher/legendary/releases/download/0.20.35/legendary_macOS_x86_64
Downloading legendary for win32 x64 from https://github.com/Heroic-Games-Launcher/legendary/releases/download/0.20.35/legendary_windows_x86_64.exe
Downloading legendary for darwin arm64 from https://github.com/Heroic-Games-Launcher/legendary/releases/download/0.20.35/legendary_macOS_arm64
Downloading gogdl for linux x64 from https://github.com/Heroic-Games-Launcher/heroic-gogdl/releases/download/v1.1.1/gogdl_linux_x86_64
Downloading gogdl for darwin x64 from https://github.com/Heroic-Games-Launcher/heroic-gogdl/releases/download/v1.1.1/gogdl_macOS_x86_64
Downloading gogdl for win32 x64 from https://github.com/Heroic-Games-Launcher/heroic-gogdl/releases/download/v1.1.1/gogdl_windows_x86_64.exe
Downloading gogdl for darwin arm64 from https://github.com/Heroic-Games-Launcher/heroic-gogdl/releases/download/v1.1.1/gogdl_macOS_arm64
Downloading nile for linux x64 from https://github.com/imLinguin/nile/releases/download/v1.1.1/nile_linux_x86_64
Downloading nile for darwin x64 from https://github.com/imLinguin/nile/releases/download/v1.1.1/nile_macOS_x86_64
Downloading nile for win32 x64 from https://github.com/imLinguin/nile/releases/download/v1.1.1/nile_windows_x86_64.exe
Downloading nile for darwin arm64 from https://github.com/imLinguin/nile/releases/download/v1.1.1/nile_macOS_arm64
Downloading GalaxyCommunication for win32 x64 from https://github.com/imLinguin/comet/releases/download/v0.1.2/GalaxyCommunication-dummy.exe
2024-08-10T14:54:34.189Z
Done downloading nile for darwin arm64
Done downloading legendary for linux x64
Done downloading GalaxyCommunication for win32 x64
Downloading comet for linux x64 from https://github.com/imLinguin/comet/releases/download/v0.1.2/comet-x86_64-unknown-linux-gnu
Downloading comet for darwin x64 from https://github.com/imLinguin/comet/releases/download/v0.1.2/comet-x86_64-apple-darwin
Downloading comet for win32 x64 from https://github.com/imLinguin/comet/releases/download/v0.1.2/comet-x86_64-pc-windows-msvc.exe
Downloading comet for darwin arm64 from https://github.com/imLinguin/comet/releases/download/v0.1.2/comet-aarch64-apple-darwin
Done downloading gogdl for darwin x64
Done downloading comet for darwin x64
Done downloading comet for win32 x64
Done downloading gogdl for darwin arm64
Done downloading legendary for darwin x64
Done downloading legendary for darwin arm64
Done downloading gogdl for linux x64
Done downloading gogdl for win32 x64
Done downloading nile for darwin x64
Done downloading nile for linux x64
Done downloading nile for win32 x64
Done downloading legendary for win32 x64
Done downloading comet for linux x64
Done downloading comet for darwin arm64
2024-08-10T14:54:39.277Z

```